_posts/2022-11-12-Aya-gsoc-final-report.md

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title: GSoC Report: "Adding a new guessing methodology for molecular systems properties in MDAnalysis"
+---
+
+
+
+## Project motivation
+
+In [MDAnalysis]molecular topologies come from various file formats and forcefields, each of which has its own rules regarding formatting and molecular properties representation. Not all molecular properties are read from files, either because they are simply not provided in the file or because they can be inferred from the nature of the molecular system or forcefield definitions (e.g., for atomic systems masses can be inferred from atom type, while in the Martini forcefield there are only three definitions of masses for its known beads). MDAnalysis had different guessing methods that can infer missing molecular properties for the [Universe]'s topology (eg. [guessing Atom type, mass, and bond]). But those methods suffered from the main issue of being so generic, which causes the guessing output to be inaccurate and not reliable for some topologies as not all topologies speak the same language. That’s why we need to develop tailored context-aware guesser classes to serve different molecular dynamics worlds, in addition, we need to establish a way to pass this context to the [Universe], so the [Universe] uses the appropriate guesser class in guessing topology attributes. 
```

Comment:
A comma is missing at the beginning of the line.

```suggestion
In [MDAnalysis], molecular topologies come from various file formats and forcefields, each of which has its own rules regarding formatting and molecular properties representation. Not all molecular properties are read from files, either because they are simply not provided in the file or because they can be inferred from the nature of the molecular system or forcefield definitions (e.g., for atomic systems masses can be inferred from atom type, while in the Martini forcefield there are only three definitions of masses for its known beads). MDAnalysis had different guessing methods that can infer missing molecular properties for the [Universe]'s topology (eg. [guessing Atom type, mass, and bond]). But those methods suffered from the main issue of being so generic, which causes the guessing output to be inaccurate and not reliable for some topologies as not all topologies speak the same language. That’s why we need to develop tailored context-aware guesser classes to serve different molecular dynamics worlds, in addition, we need to establish a way to pass this context to the [Universe], so the [Universe] uses the appropriate guesser class in guessing topology attributes. 
```